src/common/options/modern/ModernOptionsParser.cpp

```diff
@@ -233,13 +233,6 @@ void ModernOptionsParser::validate(const Values &options) const
     }
 
     /* SSL */
-    if ((options.is_set("conn-ssl-verify-peer-name") || options.is_set("con-ssl-verify-peer")) && options.is_set("conn-ssl-trust-store") == false) {
```

Comment:
hmm, typo in that code, there is missing `n` in `con-` in second or part

Reply:
yes, also noted, it was not doing the expected.